@here/harp-datasource-protocol/lib/DecodedTile.ts

```diff
@@ -87,9 +87,10 @@ export interface PathGeometry {
 
 /**
  * Attributes corresponding to some decoded geometry. It may be either a map
- * of multiple attributes or just a number with the geometry's feature id.
+ * of multiple attributes or just a string with the geometry's feature id (id numbers are
+ * deprecated).
  */
-export type AttributeMap = {} | number;
+export type AttributeMap = {} | string | number;
```

Comment:
the type of AttrinbuteMap is a bit strange. Maybe it would be better to use `JsonObject | string | number` or JsonValue ?